config/kibana.yml

```diff
@@ -111,3 +111,4 @@
 
 # Specifies locale to be used for all localizable strings, dates and number formats.
 #i18n.locale: "en"
+xpack.reporting.capture.browser.type: "chromium"
```

Comment:
Do not check local changes into kibana.yml

Reply:
woops, 🤕